src/main/java/com/exasol/adapter/dynamodb/mapping/NestedTableMappingFactory.java

```diff
@@ -0,0 +1,51 @@
+package com.exasol.adapter.dynamodb.mapping;
+
+import javax.json.JsonObject;
+
+import com.exasol.adapter.dynamodb.documentpath.DocumentPathExpression;
+
+/**
+ * This class builds {@link TableMappingDefinition}s from exasol document mapping language definitions for nested lists.
+ * It is called from {@link AbstractTableMappingFactory} when a {@code ToTableMapping} was found.
+ */
+class NestedTableMappingFactory extends AbstractTableMappingFactory {
+    private final TableMappingDefinition parentTable;
+    private final String containingListsPropertyName;
+    private final DocumentPathExpression.Builder sourcePath;
+
+    /**
+     * Creates an instance of {@link NestedTableMappingFactory}.
+     * 
+     * @param parentTable                 the table mapping the object that list mapped by this table is nested in
```

Comment:
This definition is not very clear to me.

```
the parent table that list this table as nested in table
```

or 

```
the table that map this table as a nested in
```

Reply:
I tried to clearify it. Not shure if it did help...